src/databricks/labs/ucx/queries/assessment/main/01_3_all_tables.sql

```diff
@@ -1,17 +1,31 @@
 -- viz type=table, name=Table Types, search_by=name, columns=name,type,format,storage,is_delta,location
 -- widget title=Table Types, row=1, col=3, size_x=3, size_y=8
-SELECT CONCAT(`database`, '.', name) AS name,
+SELECT CONCAT(tables.`database`, '.', tables.name) AS name,
        object_type AS type,
        table_format AS format,
        CASE
            WHEN STARTSWITH(location, "dbfs:/mnt") THEN "DBFS MOUNT"
            WHEN STARTSWITH(location, "/dbfs/mnt") THEN "DBFS MOUNT"
+           WHEN STARTSWITH(location, "dbfs:/databricks-datasets") THEN "Databricks Demo Dataset"
+           WHEN STARTSWITH(location, "/dbfs/databricks-datasets") THEN "Databricks Demo Dataset"
            WHEN STARTSWITH(location, "dbfs:/") THEN "DBFS ROOT"
            WHEN STARTSWITH(location, "/dbfs/") THEN "DBFS ROOT"
            WHEN STARTSWITH(location, "wasb") THEN "UNSUPPORTED"
            WHEN STARTSWITH(location, "adl") THEN "UNSUPPORTED"
            ELSE "EXTERNAL"
        END AS storage,
        IF(format = "DELTA", "Yes", "No") AS is_delta,
-       location
-FROM $inventory.tables
+       location,
+       CASE
+            WHEN size_in_bytes IS null THEN "Non DBFS Root"
+            WHEN size_in_bytes = 9223372036854775807 THEN "INVALID SIZE ESTIMATE"
```

Comment:
```suggestion
```
irrelevant

Reply:
Why? There's a known bug.

Reply:
I set up a cap at 10PB.
Also this error is only with parquet.
We may want to omit Parquet file altogether as we are not "deep cloning" them.
We have to figure how to handle parquet and other formats.